src/stacks-hierarchy/Root_ActiveTokenOnPhoneRequiredForFareProductScreen.tsx

```diff
@@ -54,7 +54,7 @@ export const Root_ActiveTokenOnPhoneRequiredForFareProductScreen = ({
         return;
       }
       setSaveState({saving: true, error: false});
-      const success = await toggleToken(selectedToken.id);
+      const success = await toggleToken(selectedToken.id, true);
```

Comment:
@KristianSelnas Bypass `true` is only sent while changing to mobile. Then it doesn't count to the user that change. Otherwise, will count the change.